### PR TITLE
Updating  version and adding  to the dependencies.

### PR DESCRIPTION
**Pull request by deniscavalli**
Updating `package.xml` to `format` 2 and adding `console_bridge` as dependency.

### CODE REVIEW

File: cpp_common/package.xml [hunk 1]
```diff
@@ -20,6 +20,9 @@
   <build_depend>boost</build_depend>
   <build_depend>libconsole-bridge-dev</build_depend>
 
-  <run_depend>boost</run_depend>
-  <run_depend>libconsole-bridge-dev</run_depend>
+  <exec_depend>boost</exec_depend>
+  <exec_depend>libconsole-bridge-dev</exec_depend>
```
Comment (AlexisTM) [inline cpp_common/package.xml]: This should be removed, it should be either the system installed dependency (libconsole-bridge-dev) or the catkin installed one (console_bridge)

**Review by AlexisTM — CHANGES_REQUESTED** [review]

File: cpp_common/package.xml (hunk 1 above)
Comment (deniscavalli) [reply cpp_common/package.xml]: Indeed. I will remove that.

**Review by AlexisTM — APPROVED** [review]
~REDACTED~

**Review by AlexisTM — CHANGES_REQUESTED** [review]
The change should also be applied to the CMakeLists.txt

https://github.com/TheKobiThirdParty/roscpp_core/blob/kobi/cpp_common/CMakeLists.txt

The package should be found as a catkin component & should be removed from `catkin_package(DEPENDS)`

https://github.com/TheKobiThirdParty/roscpp_core/blob/1bcf90b16a2fc129cfe6c19f03c95337d6c43b8f/cpp_common/CMakeLists.txt#L5
https://github.com/TheKobiThirdParty/roscpp_core/blob/1bcf90b16a2fc129cfe6c19f03c95337d6c43b8f/cpp_common/CMakeLists.txt#L8

**Comment by AlexisTM** [comment]
We also need to add the library linking dependency `${catkin_LIBRARIES}` https://github.com/TheKobiThirdParty/roscpp_core/blob/1bcf90b16a2fc129cfe6c19f03c95337d6c43b8f/cpp_common/CMakeLists.txt#L35

This would be "best practice" as a catkin package, but should work as is

~And add it to catkin_package as catkin package dependency `CATKIN_DEPENDS console_bridge`~ // This was already done

**Comment by deniscavalli** [comment]
Building as it is, without changing the `CMakeLists.txt` works fine. The `CMake` is already looking to the workspace first.
```sh
[build] Found '146' packages in 0.0 seconds.
Starting  >>> catkin                       
Finished  <<< catkin                        [ 0.2 seconds ]
Starting  >>> console_bridge               
Finished  <<< console_bridge                [ 0.1 seconds ]
Starting  >>> cpp_common                   
________________________________________________________________________________________________________
Warnings   << cpp_common:check /kobi/workspace/log_x64/cpp_common/build.check.001.log
CMake Warning at /home/kobi/dev/tp/src/roscpp_core/cpp_common/CMakeLists.txt:18 (message):
  console_bridge_INCLUDE_DIRS is /kobi/workspace/install_x64/include


cd /kobi/workspace/build_x64/cpp_common; catkin build --get-env cpp_common | catkin env -si  /usr/bin/make cmake_check_build_system; cd -
........................................................................................................
Finished  <<< cpp_common                    [ 0.5 seconds ]
[build] Summary: All 3 packages succeeded!
[build]   Ignored:   143 packages were skipped or are blacklisted.
[build]   Warnings:  1 packages succeeded with warnings.
[build]   Abandoned: None.
[build]   Failed:    None.
[build] Runtime: 1.1 seconds total.
kobi@kobi_x64_dev-latest:~/dev/tp$ 

```

**Review by AlexisTM — APPROVED** [review]
LGTM

**Pull request merged**